frontend/packages/form/package.json

```diff
@@ -57,7 +57,7 @@
     "access": "public"
   },
   "dependencies": {
-    "lit": "^2.0.0-rc.2",
+    "lit": "^2.0.0",
```

Comment:
Here `lit` should be moved to `peerDependencies`:
```
"dependencies": {
  "validator": "^13.6.0"
},
"peerDependencies": {
  "lit": "^2.0.0"
}
```

Reply:
thanks, done